Makefile

```diff
@@ -6,10 +6,12 @@ all:
 lint:
 	$(MAKE) -C tiny86 lint
 	$(MAKE) -C mttn lint
+	verible-verilog-lint tiny86/*.v
```

Comment:
For both of these, I'd move this logic into the `tiny86/Makefile`. Having `tiny86/*.v` here matches only:

```sh
; ls tiny86/*.v
tiny86/check.v
```

The specialized Makefile has some constants defined, like `ALL_V_WITHOUT_TESTS_OR_CODEGEN` that should handle touching all the files for you 🙂